client/utilities/supportPricing/suggestedAmounts.ts

```diff
@@ -0,0 +1,275 @@
+import type { PaidSubscriptionPlan } from '../../../shared/productResponse';
+import type { CurrencyIso } from '../currencyIso';
+import { isCurrencyIso } from '../currencyIso';
+import {
+	getSuggestedAmountsAnnual,
+	getSuggestedAmountsMonthly,
+} from './supporterPlusPricing';
+
+type SuggestedAmountsLookup = Record<
+	CurrencyIso | 'international',
+	{
+		month: (amount: number) => number[];
+		year: (amount: number) => number[];
+	}
+>;
+
+export function getSuggestedSupportAmounts(mainPlan: PaidSubscriptionPlan) {
+	const currentAmount = mainPlan.price / 100;
+	const currencyISO = isCurrencyIso(mainPlan.currencyISO)
+		? mainPlan.currencyISO
+		: 'international';
+
+	return suggestedAmountsLookup[currencyISO][
+		mainPlan.billingPeriod as 'month' | 'year'
+	](currentAmount);
+}
+
+/* 
+    ToDo: we need a more flexible way of calculating these suggested amounts, which came from a spreadsheet
+*/
+
+const suggestedAmountsLookup: SuggestedAmountsLookup = {
+	GBP: {
+		month: getGbpMonthly,
+		year: getGbpAnnual,
+	},
+	USD: {
+		month: getUsdMonthly,
+		year: getUsdAnnual,
+	},
+	EUR: {
+		month: getEuroMonthly,
+		year: getEuroAnnual,
+	},
+	AUD: {
+		month: getAusMonthly,
+		year: getAusAnnual,
+	},
+	NZD: {
+		month: getAusMonthly,
+		year: getAusAnnual,
+	},
+	CAD: {
+		month: getUsdMonthly,
+		year: getUsdAnnual,
+	},
+	international: {
+		month: getUsdMonthly,
+		year: getUsdAnnual,
+	},
+};
+
+function getGbpMonthly(amount: number) {
+	switch (true) {
```

Comment:
Why a switch here rather than just 
```javascript
if(amount <= 3) 
 return [5, 7, 10];
```
?

Reply:
I think that `if` might be better, I started with switching on the value and but the ranges were too big on the annuals